Loop/Settings/SettingsView.swift

```diff
@@ -69,4 +69,13 @@ struct SettingsView: View {
         }
         .fixedSize(horizontal: true, vertical: true)
     }
+
+    enum SettingTab: Int {
```

Comment:
Really small request: can you change `SettingTab` to `SettingsTab` since it's a single tab *within* multiple settings?

Reply:
Sure, renamed to `SettingsTab`.